### PR TITLE
Rs 2022 08 14.2

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-08-17-233720_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 1]
```diff
@@ -61,22 +71,24 @@ Below them is the crater that had become a battlefield
...
 -- Page 789, Offset 12997.
 -- 見れば、クレーターではいくつかの人影がせわしなく活動していた。
-If they looked down, they would see several figures hurrying about their work inside the crater.
+If they were to look down, they would see a handful of figures beavering away inside it.
 }
 [sha:9bbd9f9f454501d5fe6da28d9647b693c9f72085]{
 -- Page 789, Offset 12998.
 -- 災害現場の人命救助に出動したレスキュー隊、および所轄の警察から派遣された調査団だ。
-There is the rescue unit dispatched to save lives at the disaster site, as well as the investigation team dispatched from the relevant police section.
+Some are a members of a search and rescue unit dispatched to the disaster site,
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:79]: remove 'a'

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 2]
```diff
@@ -689,99 +812,115 @@ The sovereigns of the Dead Apostles. Beings considered by the Executors as the h
...
+"%{i}Tch%{/i}, I can't believe that bitch did it again.
+ The spoils of war should always be left as a tithe.
+ Is she fucking making fun of our organization's hierarchy, huh?"
 }
 [sha:99882a66c8445515ef4650c199d99a4655d1846c]{
 -- Page 849, Offset 13134.
 -- 「いやあ、そんなワケのわからない<戦利品|モノ>押しつけられても困りますけどねぇ。
-// It would be best to rephrase it to delete the pronouns
-"Weell, I wouldn't know what to do, if such incomprehensible things were loaded off to me.
+"I meeannn, I wouldn't have a clue what to do with half of that stuff it ended up being pawned off on me.
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:885]: **if** it ended up

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 3]
```diff
@@ -689,99 +812,115 @@ The sovereigns of the Dead Apostles. Beings considered by the Executors as the h
...
 }
 [sha:06fe8741a9216d109759e76fe1a5e3bcb00c5dda]{
 -- Page 849, Offset 13135.
 -- 餅は餅屋、化け物は化け物殺しにでいいじゃないッスか」
-Every man to his trades, so isn't it fine to leave monsters to the monster-killer?"
+// Not sure if there's a pithier one of this in EN.
+// 'Every man to his trade' has more of a 'stay in your lane' vibe that I think isn't present here as much
+// Alternative would be to localize mochi (We leave bread to bakers, so doesn't it...)
+// Bread analogy also amuses me because of Ciel baking
+You want good results you go to a specialist, so doesn't it make sense to leave monsters to the monster-killer?"
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:894]: localising to bread sounds good to me

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 4]
```diff
@@ -61,22 +71,24 @@ Below them is the crater that had become a battlefield
 [sha:e67efe6edeb4c986f11fe524af54f53930a56854]{
 -- Page 789, Offset 12997.
 -- 見れば、クレーターではいくつかの人影がせわしなく活動していた。
-If they looked down, they would see several figures hurrying about their work inside the crater.
+If they were to look down, they would see a handful of figures beavering away inside it.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:74]: maybe drop "inside it" or just "it". don't feel too strongly though

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 1 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt:79]: unless "units" becomes plural, this suggestion doesn't work. In the CG is there a lot of them?

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 5]
```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
-"Don't sweat it. Japanese people are passionate about their work, no? If their organization's policy states it, they'll have no choice but to obey.
+"Don't sweat it.
+ Japanese people are real hard workers, right?
+ They ain't gonna disobey if their organization's got a strict confidentiality policy.
 }
 [sha:9eddb6040a007371b45faa774a25d0af87efa18f]{
 -- Page 791, Offset 13003.
 -- よしんば話が漏れても噂止まりだ。事故現場に死体があるのは不思議じゃねえんだしな」
-Even if it leaks, it'll be nothing more than a rumor. Besides, there's nothing incredible about dead bodies being present at the scene of an accident."
+'Sides, even if anyone does let it slip, it'll just be a rumor.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:108]: does in italics

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 6]
```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
-"Well, it might not be %{i}incredible%{/i}, but isn't there something unnatural about it? Like, where'd all these bodies come from?"
+"Well, sure, it might not be %{i}unbelievable%{/i} but it's still a little %{i}unusual%{/i}, right?
+ Like, where'd all these bodies come from?"
 }
 [sha:5e902f69c2a556c008e3495fc471a5739157d5ae]{
 -- Page 793, Offset 13005.
 -- 「そりゃあオマエ、ここで発見された死体は今夜ここにいた哀れな一般人だろうが。発見された死体イコール、さっきまで生きていた人間ってのが自然だろ？」
-"Listen you, the corpses we found here are the bodies of the poor civilians who were here tonight. Dead bodies found equal the number of previously living people, capeesh?"
+"Listen dipshit, any corpses we turn up here are just the bodies of the poor unfortunate civilians who happened to be here tonight.
+ One dead body equals one human that was definitely alive earlier today, capisce?"
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:121]: people who are dead used to be alive

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 7]
```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
 -- Page 793, Offset 13006.
 -- 「げ。マジですか。そんな方便で通るんスか！」
-"Geh. Seriously? Will something so convenient work?"
+"%{i}Geh%{/i}. Seriously? Will something that convenient work?"
 }
 [sha:712c98a29b2debbd637b7ef7d84c82367de45e7d]{
 -- Page 794, Offset 13007.
 -- 「そこを通すんだよ。いやあ、公園だったのが幸いしたぜ。無関係な人間が山ほどいたコトの理由になる」
-"It will. Ahh, we're lucky it happened in a public park. We can use the fact that there were lots of unrelated people present."
+"Oh it'll work.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:131]: Oh, it'll

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 8]
```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
 [sha:260e6fea6049d4e3def888f6142f9558abb09066]{
 -- Page 795, Offset 13008.
 -- 「理由さえあればあとは力仕事だわな。何の為にうちのジジイが権力バラまいてると思ってやがる。押さえるのは<上|トップ>だけでいいんだよ。下っ端や真ん中が不審がっても関係ねえ。最後にまとめる人間が握りつぶせばそれでおしまいだ」
-"As long as we have a pretext, the rest is just a matter of manpower. Why do you think my old man's handing out power for? The only people we have to keep in check are the ones on top. It doesn't matter if the underlings or those in the middle get suspicious. As long as the people in charge don't let anything through, it'll go nowhere."
+// Reading 散播く as _wielding_ political power rather than _disseminate_ (i.e, 散播き政治)
+// since that seems to make more sense in ctx
+"So long as we have a cover story, the rest is just a matter of legwork.
+ Why d'ya think my old man's been leaning on people left and right?
+ All you gotta do is put pressure on the guys at the top.
+ Doesn't make a fat bit of difference if the small fry or middle managers get suspicious.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:143]: if a small fry or a middle manager gets suspicious
if small fries or middle managers get suspicious

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 9]
```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
+ You're still young but you're already gunning it for the fast track aren't you, young master?
+ Both in terms of your social status and your mindset.
+ I mean, don't you ever want to act your age and play around once in a while?"
 }
 [sha:6b5a46592fca4924d9eda81d955c113f1bc5c10d]{
 -- Page 797, Offset 13011.
 -- 「あるかンなもん。ガキってのは基本つまらねえもんだって忘れたのか30歳？　オラ、無駄話してねえで監督してきやがれ。ヤバめな証拠は隠滅しとけよ」
 // He says 30歳, thirty year-old. Just imagine that I thought about "30 year old boomer" for a second haha
-"No fucking way. Have you forgotten that brats are all dull, you 30-year-old-geezer? Quit it with your nonsense and go supervise. And make sure to destroy any dangerous evidence."
+"Absofucking%{i}lutely%{/i} not.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:168]: Abso-fucking-%{i}lutely%{/i} not.

might look completely fucked with italics but /shrug

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 10]
```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
 -- Page 797, Offset 13012.
 -- 「へーい。上司の命令とあればよろこんでー」
-"Yeah. If those are my boss's orders, with pleasure."
+"Alrighty. If those are my orders, then with pleasure."
 }
 [sha:a5be56357297b2f26e024694d5b19b0989a57747]{
 -- Page 798, Offset 13013.
 -- 「……アレでもちっと緊張感があれば使えるんだがな。あの捨て鉢さはなんだ、若い頃に挫折でもしたのかカリウス？」
-"...If only he had some sense of urgency, then he'd be more useful. To be so low-spirited, did he break down when he was young or something? What do you think, Karius?"
+"...That guy would really come in handy if he had any fucking sense of urgency at all.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:181]: maybe any in italics

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 11]
```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
 -- Page 799, Offset 13014.
 -- 「私にはとんと。安藤の私生活に興味はありませんので。
-"I haven't the slightest idea. I don't have any interest in Andou's private life.
+"I haven't the slightest idea. Andou's private life holds no interest for me.
 }
 [sha:4d5e1406f183d7f3cc5b7de59cefb742bdfb0098]{
 -- Page 799, Offset 13015.
 -- ３年前、悪徳司祭の内部告発を決意したものの、最後の最後で金に目が眩んだ恋人に裏切られ、故郷を追われ、失意の果てに父方の祖国であるこの国に島流しになった、という事ぐらいしか。
-I only know that he decided to blow the whistle on a corrupt priest, but at the last minute, was betrayed by his lover who had been blinded by the promises of riches, was driven out from his hometown, and was, at the depths of despair, exiled to Japan, his paternal native country.
+All I know is that three years ago he decided to blow the whistle on a corrupt presbyter,
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:193]: I'm hesitant to add another comma to _the_ run-on sentence of the year, but
> years ago, he decided

Maybe just reflow it to something like:
> All I know is that three years ago, he decided to blow the whistle on a corrupt presbyter. He got betrayed in the final hour though by his lover, who had been blinded by greed. After that, he was driven from his hometown before, as the final act in his tale of adversity, he was exiled to his father's homeland―――to this country.

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 12]
```diff
@@ -172,12 +217,13 @@ Also, it seems that after paying back 30,000 yen to a bank last month, he took o
...
 -- 「おう、仲良くな。んじゃあそろそろ報告でも聞いとくか。
-"Yeah, get along well, okay? Well, it's time to hear the report then.
+"Yeah, get along well, okay?
+ Now then, 'bout time we hear that report.
 }
 [sha:e4dd99a4baa0e0f2e4a0070129cbc7b03d7bb9c2]{
 -- Page 801, Offset 13020.
 -- あの使えない方の女、通していいぜ」
-You can let that useless woman through."
+You can let that female waste of space though."
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:226]: through

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 13]
```diff
@@ -217,63 +263,73 @@ Her surprise is completely understandable.
...
 -- Page 804, Offset 13029.
 -- 聖堂教会の頂点である法王と、
-The Pope, the pinnacle of the Holy Church.
+The Pope is the pinnacle of the Holy Church.
 }
 [sha:cf570b2b5da72eed3a990720850659578e49f0d1]{
 -- Page 804, Offset 13030.
 -- その法王を支える128人の枢機卿。
-As well as the 128 Cardinals assisting him.
+Assisting him are 128 cardinals.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:276]: as much as I love the power of 2 cardinal count, i think there's way more, mister

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 14]
```diff
@@ -217,63 +263,73 @@ Her surprise is completely understandable.
...
 -- Page 805, Offset 13034.
 -- 血族主義ではなく実力主義。
-// 血族主義 is actually not an real compound word, though readily understandably. DeepL proposed bloodocracy, which is pretty nice. I learned that the word nepotism actually find its roots in the practice of Cardinals appointing nephews to important positions, which is perfect here, except that nepotism today also works for friends. So blood nepotism.
-This isn't blood nepotism but meritocracy.
+// G: 血族主義 is actually not an real compound word, though readily understandable.
+// G: Bloodocracy sounds cool, but isn't really a word in EN.
+// G: I learned that the word nepotism actually find its roots in the practice of Cardinals appointing nephews to important positions, which is perfect here, except that nepotism today also works for friends.
+// R: Rendered as dynastocracy here, which is tenuously a word that's in use,
+// but is more arguable than hemocracy or whatever and keeps the common ~主義 nicely.
+It isn't a dynastocracy so much as a meritocracy.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:305]: that _is_ a dynasty, dawg

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 15]
```diff
@@ -294,47 +352,54 @@ His one and only flaw is his love of women.
...
+// Reading 遅咲き is late_comer_ (young out of his brothers) rather than late _bloomer_ (he seems to have bloomed very early)
+Cardinal Laurentis holds Mario――who is ostensibly his grandson in name only――in high regard,
+ and there are whispers that he's likely to come from behind to claim the title of successor.
 }
 [sha:beecb60f3a4432254c78b4b903bcfe187b543397]{
 -- Page 810, Offset 13044.
 -- ベスティーノという姓はラウレンティス枢機卿がまだ片田舎に住んでいた頃の旧姓である。20人を超える養子……いや、兄たちを差し置いて、この姓を与えられた少年がどれほど期待されているかなど、信徒であれば説明されるまでもない。
-The surname Bestino is the old family name of Cardinal Laurentis, back from where he was still living in the remote countryside. All the faithful know very well just how much is expected of the young boy who was granted this surname, over more than twenty other adopted sons... no, over more than twenty of his brothers.
+The surname Bestino is Cardinal Laurentis's old family name, a memento from when he was still living in the backcountry.
+ It's obvious to any member of the clergy just how high Laurentis's expectations must be for this boy,
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:363]: drop comma even though it will result in a MEGA SENTENCE
_maybe_ comma after "surname".

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 16]
```diff
@@ -344,18 +409,21 @@ Why has such a boy come to this island nation all alone...?
...
 -- Page 813, Offset 13053.
 -- で、でも……あの、枢機卿のお孫さんにしては、その……若すぎる……ような？」
-I-It's just... Umm, for the grandson of a Cardinal, you're... sort of... too young?"
+I-It's just... Umm, you seem... well... you know... too young to be the Cardinal's grandson?"
 }
 [sha:43363c4ec437aca7817ae62d2899e9443c768d5f]{
 -- Page 814, Offset 13054.
 -- 「そりゃあ若ぇだろ。ジジイが百歳超えてから愛人に産ませたガキだからな。今年で12だ、おそれいったか。しかもまだ弟がいるって話だ。
-"Sure, I'm young. I'm the child of a mistress the old man impregnated when he was more than a hundred years old. I'm twelve this year, surprised? And what's more, I even have some younger brothers.
+"You bet I'm goddamn young.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:417]: Goddamn right I'm young.

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 17]
```diff
@@ -370,109 +438,125 @@ W-What are you saying...?!"
...
-By 'the beginning', when exactly was that... sir?"
+When you say 'opening act', which part was that exactly... sir?"
 }
 [sha:90dca1eeb69a2b3e5ea14bb2383ff3a59b9fe17e]{
 -- Page 818, Offset 13063.
 -- 「テメェが逃げ出したところからだ。いい判断だったな代行者ノエル？　おかげで命があるワケだ。三流は三流らしく、テメェの命を最優先ってなァ？
-"Around when your dumb ass ran away. That was a good decision, Executor Noel, huh? Thanks to that, you're still alive. A third-rate doing as third-rates do and prioritizing her own life, right?
+"It was the part where you turned tail and legged it, asshat.
+ Good thing you did too, right, Executor Noel?
+ I mean thanks to that you're still in one piece.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:470]: that, you're

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 18]
```diff
@@ -370,109 +438,125 @@ W-What are you saying...?!"
...
+ You can always trust a third rate piece of shit to give their life top priority, can't ya?
 }
 [sha:f58f44656ea28b85efab34432cd17d18ca153143]{
 -- Page 818, Offset 13064.
 -- 逃げ足だけは見所あるよ、オマエ。10年近く使い捨ての狗ころやってるヤツは筋金入ってやがる。オレが法王庁に帰り次第、テメェは首だ。修道院送りじゃねえぞ。ギロチン送りの方な」
 // 首を切る meaning to fire someone and to behead someone.
-The only thing you're good at is running away. Those who've served as disposable dogs for almost ten years should have some backbone. As soon as I return to the Vatican, I'll have your head. And I don't mean I'll send you to a monastery. I mean I'll send you to the guillotine."
+Only thing you're any goddamn good at is running away.
+ You'd think that after being a disposable attack dog for ten years you'd have grown a fucking backbone.
+ The second I get back to the Vatican I'm gonna have your head.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:479]: Vatican, I'm

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 19]
```diff
@@ -482,52 +566,64 @@ I'm not lying. I can even submit myself to the questions of the Vatican! As Depu
...
 -- 自らの失言に気がつくノエルだが、こうなっては隠し通せない。
-Noel realizes her own slip of the tongue, but knows she can't cover it up now that it's come to this.
+Noel realizes that she's said too much, but she can't keep hiding it at this point.
 }
 [sha:2f1191075f663495cd357b18b848347e1a341f11]{
 -- Page 828, Offset 13085.
 -- 彼女は同僚である代行者と行ったここ一ヶ月の調査内容を、本国からやってきた司祭代行に報告した。
-She informs the Acting Presbyter, who had just arrived from his home country, regarding the content of the investigation she and her partner conducted over the past month.
+She fills in the Acting Presbyter, who has just flown in from her homeland,
+ on the details of the investigation she and her partner had conducted over the past month.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:601]: investigation that she

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 20]
```diff
@@ -482,52 +566,64 @@ I'm not lying. I can even submit myself to the questions of the Vatican! As Depu
...
 -- ノエルの報告は10分ほどで終わった。
-She concludes her report after speaking for about ten minutes.
+It takes her around ten minutes to finish her report.
 }
 [sha:d22a7f25258fdc639f8f287ad4bbf00e366cbb94]{
 -- Page 829, Offset 13087.
 -- 簡潔に、この調査の責任は自分にはなく、あくまで同僚にある、と主張しながら。
-Briefly, while insisting the responsibility for the investigation fell, not on her, but entirely on her partner.
+Much of it boils down to her insisting that she wasn't the one who instigated the investigation,
+ and that any responsibility for it lies squarely with her partner.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:612]: Noel my beloved

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 21]
```diff
@@ -482,52 +566,64 @@ I'm not lying. I can even submit myself to the questions of the Vatican! As Depu
...
+"You can't score any brownie points off of unconfirmed information, you need something solid to back it up.
 }
 [sha:7dec19c8de847bab5581bc316db7e28dc833958d]{
 -- Page 830, Offset 13089.
 -- つまりはアレだ。テメェらも独断で動いてたワケか。結構結構、それなら潰し合う必要はねぇ。協力し合えそうじゃねえか、代行者ノエル？」
-So that's it. You've been working without consulting anyone. Good good. In that case, there's no need for us to quarrel. Seems like we can cooperate, huh, Executor Noel?"
+So that's your angle.
+ You fuckers have also been acting on the sly, haven't you.
+ That's great, that's real great.
+ In that case we don't need to be at each others throats now do we.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:625]: throats, now

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 22]
```diff
@@ -537,62 +633,80 @@ So that's it. You've been working without consulting anyone. Good good. In that
...
 -- Page 835, Offset 13097.
 -- 「ああ、互いに助け合っていこうぜ。
-"Ahhh, let's say that we're helping each other out.
+"Yeah, we're in a position to help each other out.
 }
 [sha:a41da7a767d69b428eb377a0f69fbf59f94cb5d5]{
 -- Page 835, Offset 13098.
 -- とは言ってもコトが終われば立場も元通りだ。<後々|あとあと>いい生活がしたいってんなら尻でも振りまくるんだな。
-That being said, when everything's over, our positions will return to what they were originally. If you want to have a nice life in the future, then shake your ass.
+Don't get any ideas though, soon as this is over it's back to business as usual.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:684]: over, it's

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 23]
```diff
@@ -689,99 +812,115 @@ The sovereigns of the Dead Apostles. Beings considered by the Executors as the h
...
+// Reading this as Noel's upcoming 打ち合わせ because Mario just fucks off on his own after this
+"I had Karius send her back. She's got a briefing she's gotta get to anyway.
 }
 [sha:40ad5f565906e035fee67b1a8d316c5633db03a1]{
 -- Page 847, Offset 13128.
 -- 埋葬機関の方は話にならねえが、あの女は分かりやすい。ニンジンぶらさげりゃあ死ぬまで走るタマだ。
 // I'm not sure what 走るタマ is. Is it like 球が走る in baseball or 霊?
-It's no use talking about the Burial Agency, but that girl is easy to understand. Dangle a carrot in front of her, and she'll run until she dies.
+It's no use talking to the one from the Burial Agency, but that girl's easy to read.
+ Dangle a carrot in front of her, and she'll run until she dies.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:856]: drop comma

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 3 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt:894]: i wouldn't mind that localization but this line is fine as-is imo

**Review by Avuxo — APPROVED** [review]
fucking otsukare, jesus christ this file

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 11 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt:193]: I'm happy to add another comma, I think we should keep this sentence as heinously long as possible since it's basically a joke

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 13 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt:276]: Just checking this is a meta comment not a TL comment? You got me worried

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 8 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt:143]: Hmm, not sure about this one. I feel like it should be plural here (doesn't matter even if a bunch of lesser folk are suspicious) but 'small fries' doesn't read right to me as the plural form of small fry. I think it's just a defective noun.

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 13 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt:277]: meta-commentary

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 8 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt:143]: Seems like [both are correct](https://www.merriam-webster.com/dictionary/small-fry) but I really, really don't care in this case
